index.d.ts

```diff
@@ -289,6 +291,16 @@ declare module "leap-core" {
     tendermintAddress: string;
   }
 
+  export type Signer = {
+    signTx: (tx: Transaction) => Promise<Tx<any>>;
```

Comment:
Leap transaction in parameter? In that case, you can define it like this:
```
signTx: <TxType extends Type>(tx: Tx<TxType>) => Promise<Tx<TxType>>;
```

Reply:
good catch! But why `<TxType extends Type>` is needed?